types/Videoinstruction/type.d.ts

```diff
@@ -0,0 +1,50 @@
+import type { IPagination } from "~/types/Base/IPagination";
+import type { IVideoinstruction } from "~/types/Videoinstruction/IVideoinstruction";
+
+type VideoinstructionId = number;
+
+export interface IVideoinstructionPayloadBase {}
+export interface IVideoinstructionResponseBase extends IVideoinstruction {}
+
+export interface IVideoinstructionPayloadCreate
+  extends IVideoinstructionPayloadBase {
+  title: string;
+  platform: number | undefined;
+  video_url: string;
+}
+
+export interface IVideoinstructionResponseCreate
+  extends IVideoinstructionResponseBase {}
+
+export interface IVideoinstructionPayloadUpdate
+  extends IVideoinstructionPayloadBase {
+  id: number;
+  title: string;
```

Comment:
Тут все поля кроме id необязательные.

```suggestion
  title?: string;
```